interview.md

```diff
@@ -5,47 +5,93 @@
 This section describes who the project would serve and why a language might be a
 good way to meet their needs.
 
+    The user is someone who needs to manage heat pumps in different homes.
+    A language might be a good way to manage individual parameters of different
+    heat pumps.
+
 ### What's the need?
 
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help
 them?_
 
+    The need is met as many different parameters need to be taken into account
+    before installing a heat pump. I would be helping people like my brother who
+    need to install heat pumps to be as energy efficient as possible. Their
+    experience currently is that they need to take different temperatures, different
+    readings from data not publicly available, and analyze this data to determine
+    when best to have heatpumps running. This experience might be easier if
+    a DSL was used to input these parameters and output automatically the most
+    efficient and best parameters.
+
 ### Why a language?
 
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
 
+    A DSL is appropriate because it is specific to a domain and would
+    eliminate human error from analyzing data. It would also speed up
```

Comment:
It would move where the human error would occur. They could still mess up implementing with the DSL technically